src/Event/Event.php

```diff
@@ -312,7 +312,7 @@ public function updateLocation(LocationId $locationId): void
 
         if (!$locationId->isNilLocation() && $this->attendanceMode === AttendanceMode::online()->toString()) {
             throw new UpdateLocationNotSupported(
-                'Instead of passing the real location for this online event, please update the attendance mode to offline or mixed.'
+                'Cannot update the location of an online event to a real location. Set the attendanceMode to mixed or offline first.'
```

Comment:
Maybe `real location` should be changed to `physical location`. I think I have seen that used inside the API docs.

Reply:
Yeah I made the same remark in another PR now that I think about it, not sure where though. But doesn't matter, physical location is probably clearer because "real" can mean many things, could also be some kind of property on the API for all the integrator knows.